### PR TITLE
Add true lag metric to async-indicator

### PR DESCRIPTION
**Pull request by sravfeyn**
The current lag metrics includes the indicators that failed 20 times (max retry count), so it's not strictly accurate. `unsuccessful_attempts` indicator is also faulty, since when last 100 indicators are in failed state, it keeps reporting it as 2k (20*100) always as it is now, so I am adding an indicator to track how many have fully failed after max retries.

@calellowitz

### CODE REVIEW

File: corehq/apps/userreports/tasks.py [hunk 1]
```diff
@@ -322,16 +323,16 @@ def queue_async_indicators():
...
+        2. have failed less than ASYNC_INDICATOR_MAX_RETRIES times
         This task quits after it has run for more than
         ASYNC_INDICATOR_QUEUE_TIME - 30 seconds i.e 4 minutes 30 seconds.
         While it runs, it clubs fetched AsyncIndicators by domain and doc type and queue them for processing.
     """
     start = datetime.utcnow()
     cutoff = start + ASYNC_INDICATOR_QUEUE_TIME - timedelta(seconds=30)
     retry_threshold = start - timedelta(hours=4)
-    # don't requeue anything that has been retried more than 20 times
-    indicators = AsyncIndicator.objects.filter(unsuccessful_attempts__lt=20)[:settings.ASYNC_INDICATORS_TO_QUEUE]
```
Comment (sravfeyn) [inline corehq/apps/userreports/tasks.py:334]: @calellowitz 20 seems too high, any idea if we can reduce this to say 5?

**Review by calellowitz — APPROVED** [review]
Haha, I was just thinking about doing something similar today. This looks good. I do think the other one is worth keeping around, but having both is good.

As far as retries, I think 20 has minimal harm so its worth keeping it high since the consequences of maxing out are so bad.

**Review by snopoke — COMMENTED** [review]
+1 to removing the old lag / using the same name with the correct value.

**Comment by sravfeyn** [comment]
@snopoke I think Cal meant that it's good to keep both of them, since it's good to know the oldest regardless of whether it failed.

**Comment by sravfeyn** [comment]
The less failures there are the more alike these two metrics will look.

**Pull request merged**